src/pygeon/grids/grid.py

```diff
@@ -290,3 +290,27 @@ def compute_subvolumes(
             )
         else:
             return self.face_nodes @ div_by_nodes_per_face @ sub_simplices
+
+    def compute_opposite_nodes(self, recompute=False) -> sps.csc_matrix:
```

Comment:
even if the argument recompute is quite self explanatory, I would add the Args section to be consistent with the standard documentation